selfdrive/controls/lib/lateral_planner.py

```diff
@@ -103,6 +103,7 @@ def publish(self, sm, pm):
     plan_send.valid = sm.all_alive_and_valid(service_list=['carState', 'controlsState', 'modelV2'])
 
     lateralPlan = plan_send.lateralPlan
+    lateralPlan.frameId = sm['modelV2'].frameId
```

Comment:
let's use the logMonoTime like longitudinalPlan does

Reply:
Use translations instead? I think this is worse